marimo/_output/formatters/altair_formatters.py

```diff
@@ -31,31 +29,55 @@ def register(self) -> None:
         def _show_chart(chart: altair.Chart) -> tuple[KnownMimeType, str]:
             import altair as alt
 
+            # Try to get the _repr_mimebundle_ method from the chart
+            # If its HTML, we want to handle this ourselves
+            # if its svg, vega, or png, then we want to pass that instead
+            # because that means the user has configured the that renderer
+            mimebundle = {}
+            try:
+                mimebundle = chart._repr_mimebundle_() or {}
+            except Exception:
+                pass
+
+            # Handle where there are multiple mime types
+            if len(mimebundle) > 1:
+                # If its a list, return the first one
+                return (
+                    "application/vnd.marimo+mimebundle",
+                    json.dumps(mimebundle),
+                )
```

Comment:
The comment (`if it's a list, return the first one`) seems different than the code?

Reply:
thanks, fixed